src/nix/describe_derivation.nix

```diff
@@ -31,6 +31,17 @@ in
   name = targetValue.name;
   parsed_name = (builtins.parseDrvName targetValue.name);
   attribute_path = targetAttributePath;
+
+  src =
+    if targetValue ? src.gitRepoUrl && targetValue ? src.rev
+    then
+      {
+        git_repo_url = targetValue.src.gitRepoUrl or null;
```

Comment:
Not that it's important, is it possible to reach the `or null` codepath here, given the test condition of the `if`? I feel like either one of the attribute isn't defined and we end up with `src = null`, or both are defined.